File: DrawUIBezierPathFitToUIViewDemo/DrawUIBezierPathFitToUIViewDemo/ViewController.m
//
//  ViewController.m
//  DrawUIBezierPathFitToUIViewDemo
//
//  Created by Mossuru777 on 2014/11/14.
//  Copyright (c) 2014 Mossan. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];

	// set TrainView border
	for (UIView *subView in self.view.subviews) {
		subView.layer.borderWidth = 0.5;
		subView.layer.borderColor = [[UIColor grayColor] CGColor];
	}
}

@end

File: Demo/DrawUIBezierPathFitToUIViewDemo/ViewController.swift
//
//  ViewController.swift
//  DrawUIBezierPathFitToUIViewDemo
//
//  Created by Mossuru777 on 2018/01/24.
//  Copyright © 2018年 Mossan. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    override func viewDidLoad() {
        super.viewDidLoad()

        // set TrainView border
        for subView: UIView in self.view.subviews {
            subView.layer.borderWidth = 0.5
            subView.layer.borderColor = UIColor.gray.cgColor
        }
    }
}
